docs/synapse/userguides/ug013_storm_ref_lift.rst

```diff
@@ -0,0 +1,27 @@
+Storm Reference - Lift Operators
+================================
+
+The operators below can be used to retrieve a set of nodes from a Synapse hypergraph based on a set of specified criteria.
+
+All of these operators are defined in storm.py_.
+
+See the `Storm Syntax Conventions`__ for an explanation of the usage format used below.
+
+Where specific query examples are given, they are commonly provided in pairs using operator syntax followed by the equivalent macro syntax (if available).
+
+lift()
+------
+Todo
+
+guid()
+------
+Todo
+
+alltag()
+--------
+Todo
+
+.. _storm.py: https://github.com/vertexproject/synapse/blob/master/synapse/lib/storm.py
+
+.. _conventions: ../userguides/ug011_storm_basics.html#syntax-conventions
+__ conventions_
```

Comment:
This page looks good to me as long as the PRs will be done in another PR (and weren't intended to be done in this one)